src/redux/categories/Categories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+
+const initialState = [];
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
+
+const statusReducer = (state = initialState, action = {}) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return ['Under construction'];
+    default:
+      return state;
+  }
+};
+
+export const Categories = () => (
+  <section>
+    <button type="button">
+      Check status
+    </button>
+  </section>
+);
```

Comment:
- [ ] The same thing for the `Categories` component. It would be better to create a file `Categories.js` in the /src/components folder and add the Categories component there so your repo is well organized.